chia/wallet/cat_wallet/cat_wallet.py

```diff
@@ -54,11 +54,14 @@
 from chia.wallet.wallet_coin_record import WalletCoinRecord
 from chia.wallet.wallet_info import WalletInfo
 
+if TYPE_CHECKING:
+    from chia.wallet.wallet_state_manager import WalletStateManager
+
 # This should probably not live in this file but it's for experimental right now
 
 
 class CATWallet:
-    wallet_state_manager: Any
+    wallet_state_manager: "WalletStateManager"
```

Comment:
Also add `from __future__ import annotations` to the top of the file.  https://gist.github.com/altendky/7c423a63d5091b2ccf2c418624317f93#forward-references

```suggestion
    wallet_state_manager: WalletStateManager
```